src/main/ngapp/accio/src/app/editor/editor.component.ts

```diff
@@ -42,12 +43,12 @@ export class EditorComponent implements OnInit {
   scaleFactor: number;
   originalImageData: ImageData;
   tolerance: number;
+  tint:number;
```

Comment:
this is used to set the globalAlpha so I think it is better to call this alpha or maskAlpha instead of introducing a new word tint because I immediately wondered what it was since I didn't expect the word.